packages/bootleg-tokens/contracts/ISharedRoyaltyTokenEnumerable.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.5.3;
+
+/**
+* @title Shared Royalty NFT Enumerable interface for franchisors
+*/
+contract ISharedRoyaltyTokenEnumerable {
```

Comment:
What about something akin to? 

```sol
    /// @notice Enumerate NFTs assigned to an owner
    /// @dev Throws if `_index` >= `balanceOf(_owner)` or if
    ///  `_owner` is the zero address, representing invalid NFTs.
    /// @param _owner An address where we are interested in NFTs owned by them
    /// @param _index A counter less than `balanceOf(_owner)`
    /// @return The token identifier for the `_index`th NFT assigned to `_owner`,
    ///   (sort order not specified)
    function tokenOfOwnerByIndex(address _owner, uint256 _index) external view returns (uint256);
```

in our case, I suppose it would be `tokenOfFranchisorByIndex(address _franchisor, uint245 _index)`? 

Be able to iterate all of the tokens that a user is a franchisor of.